docs/js.md

```diff
@@ -413,3 +390,117 @@ mdocVariables := Map(
   "js-batch-mode" -> "true"
 )
 ```
+### Remapping imports at link time.
+
+It may be the case, that the library you wish to depend on, is a facade to some NPM dependancy. Mapping the import offers an alterative to bundling, by resolving the dependancy out of a CDN. We set mdoc to output ESModules (see `ModuleKind`), and load the dependancies directly in the browser.
```

Comment:
in English dependent sentences are not separated by commas, unlike some other languages. so, "it ... case that ... on is ...".

also, "dependency" is not spelled with an "A", used twice here.